website/src/_posts/2017-02-13-stateful-processing.md

```diff
@@ -560,7 +560,8 @@ class ModelStatefulFn(beam.DoFn):
     previous_prediction = previous_pred_state.read()
 
     new_prediction = model.prediction(event)
-    model_state.add(event)
+    model.update(event)
+    model_state.write(model)
```

Comment:
`CombiningValueRuntimeState` is the interface here, and it does not have a `write` method. The method `add` is correct here.
Furthermore, we're assuming that `ModelFromEventsFn` executes the `model.update(event)`. You could change this to have a `BagStateSpec`, and so you would run:

```
model.update(event)
model_state.clear()
model_state.add(model)
```

Or we could keep it as it was. What do you prefer?